pkg/pool-weighted/contracts/WeightedPoolProtocolFees.sol

```diff
@@ -177,6 +177,11 @@ abstract contract WeightedPoolProtocolFees is BaseWeightedPool, ProtocolFeeCache
         return rateProduct;
     }
 
+    /**
+     * @notice Returns the amount of BPT to be minted to pay protocol fees on yield accrued by the Pool.
```

Comment:
We don't usually use notice for internal functions. Maybe just @dev for the whole thing? (Same as below)

Reply:
I quite like to have both anyway as the presence of a `@dev` tag means that there's a gotcha to the function to be aware of. I've removed this though.